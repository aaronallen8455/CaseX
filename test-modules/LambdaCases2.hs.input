main :: IO ()
main = pure ()

test :: Bool -> Bool -> Bool
test = \cases
  _ SPLIT -> False
