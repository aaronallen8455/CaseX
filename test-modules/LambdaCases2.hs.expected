main :: IO ()
main = pure ()

test :: Bool -> Bool -> Bool
test = \cases
  _ False -> False
  _ True -> False
